adobe/spectrum-web-components
# fix(coachmark): add "step-count" slot for custom/internationalized pagination content

## Description

This PR adds the ability for users to customize the separator between the current step and the total steps in the YourComponent component. By default, the separator is 'of', but users can now specify a custom separator using the separator attribute.

### Changes Made
Added a new separator property to the YourComponent component to allow customization of the separator.
Updated the component's template to use the separator property for the separator between the current step and the total steps.




## Related issue(s)



- #4213 

## Motivation and context



## How has this been tested?




    1. Go [here](https://feature-coachmark-localization--spectrum-web-components.netlify.app/storybook/?path=/story/coachmark--custom-separator)
    2. Use the separator attribute to specify a custom separator.
    3. Verify that the component renders with the custom separator between the current step and the total steps.
    1. Go here
    2. Do this

## Screenshots (if appropriate)
![Screenshot 2024-03-26 at 4 23 54 PM](https://github.com/adobe/spectrum-web-components/assets/8790510/379ad2ca-60e9-43c9-b7a5-56646819e606)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
